lib/Core/SymbolicError.h

```diff
@@ -157,6 +161,10 @@ class SymbolicError {
 
   double getPathProbability() const { return pathProbability; }
 
+  void addErrorCondition(ref<Expr> error) {
```

Comment:
@Himeshi I prefer renaming of this to addErrorConstraint for uniformity with the rest of the KLEE code.